### PR TITLE
Feature Portfolio Header and Headline

### PR DESCRIPTION
**Pull request by BlesslinJerishR**
## In this Branch
- Add Linter
- Add README
- Add HTML
- Add CSS
- Add Header
- Add Headline
- Add Social Icons
- Add Flexbox
- Add Background Illustration
- Add Mobile View Background
- Add Live Demo - will work once merged !
- Fixed Mobile Responsive
- Fixed font size
- Add Favicon
- Add My Recent Work Section
- Fixed hamburger menu size bug
- Fixed background image size bug

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <img
+        id="menu"
+        src="https://img.icons8.com/external-tal-revivo-regular-tal-revivo/96/null/external-mobile-application-hamburger-menu-setting-interface-basic-regular-tal-revivo.png"
+      />
+    </nav>
+    <img id="i1" src="imgs/i1.png" alt="illustration" />
+    <section id="content-1">
+      <h1>
+        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
+        <span class="clr-green"> I'm a Software <br />Developer</span>
```
Comment (joel-kalema) [inline index.html]: Good job working on the mobile first

- [ ] Kindly make sure your website looks exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=0ECBEqbggTjVdj0j-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Your%20website%20should%20look%20exactly%20like%20the%20design%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 😃 

-  the `background image`
- the `icons` color. It should be `#505F79`  
- the `<p>` color. It should be `#172B4D`
- kindly, make sure you added this button
 btn
- Also make sure you fixed the size of the `menu` button 
menu



> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |mobile01| |mobile| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @BlesslinJerishR 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
```
Comment (joel-kalema) [inline README.md]: - [ ] Kindly, make sure you removed the logo to make your README more profetional

File: index.html (hunk 1 above)
Comment (BlesslinJerishR) [reply index.html]: Hi Sir, @joel-kalema  In my computer the mobile version the code didn't break , may I know which breakpoint px you checked my code ?

File: README.md (hunk 2 above)
Comment (BlesslinJerishR) [reply README.md]: I Removed the Logo sir.

File: index.html (hunk 1 above)
Comment (joel-kalema) [reply index.html]: ohh, Hello @BlesslinJerishR , sorry for being late, but for this question, I used `375px` as it was [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Minimum%20mobile%20screen%20size%20should%20be%20375px)
to check it well, you may `click` on the button below two times 
mob

File: index.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <img
+        id="menu"
+        src="https://img.icons8.com/external-tal-revivo-regular-tal-revivo/96/null/external-mobile-application-hamburger-menu-setting-interface-basic-regular-tal-revivo.png"
+      />
+    </nav>
```
Comment (nahnahdev) [inline index.html]: - [ ] Great job adding the hamburger menu on the right side of the design the previous reviewer suggested. However, it currently looks bigger than the Figma design. I will suggest that you reduce the size of the hamburger menu so it looks similar to the Figma design :100: 
- Your Design
image

- The Figma Design
image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      </div>
+      <img
+        id="d"
+        src="https://img.icons8.com/external-dreamstale-lineal-dreamstale/32/null/external-down-arrow-arrows-dreamstale-lineal-dreamstale-2.png"
+      />
+    </section>
+    <section id="content-2">
+      <h1 class="clr-blue">My Recent <br />Works</h1>
+      <img src="imgs/Indicator.png" alt="indicator" />
+    </section>
```
Comment (nahnahdev) [inline index.html:40]: - [ ] Lastly, the background image doesn't show properly in your design as it was in the Figma design, and just as the previous reviewer mentioned, please ensure that you have the right background image so your design looks similar to the original design :+1: 
- Your Design
mobile (1)
- The Figma Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @BlesslinJerishR , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- No linter errors :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (BlesslinJerishR) [reply index.html]: Fixed Thank you

File: index.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+    <section id="content-1">
+      <h1>
+        <span class="clr-blue">Hey there. <br />I'm Blesslin <br /></span>
+        <span class="clr-green"> I'm a Software <br />Developer</span>
+      </h1>
+      <p class="content-1-p">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
```
Comment (amejid) [inline index.html:28]: - [ ] [OPTIONAL] I highly suggest you check the `font-size` of the text on your page. It's looking big when compared to the Figma design. ✍️

**Review by amejid — APPROVED** [review]
Hi @BlesslinJerishR, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**